docs/tools/vdb_table/data/apachecassandra.json

```diff
@@ -149,27 +149,31 @@
     "comment": ""
   },
   "github_stars": {
-    "value": 8410,
+    "value": 8411,
     "unlimited": false,
     "source_url": "https://github.com/apache/cassandra",
-    "comment": ""
+    "comment": "",
+    "value_90_days": 0
   },
```

Comment:
again, here Cassandra's overall repo is not the one we'd like to pull the stars info from (as discussed with @svonava-superlinked), as we aim to compare vector database popularity through this. We need to figure out how to represent this case.